client/src/views/Conversations.tsx

```diff
@@ -18,12 +53,12 @@ const Conversations: React.FC<ConversationsProps> = (props: ConversationsProps):
             <p>View, chat, and manage your conversations.</p>
           </div>
           <div className='conversations-list'>
-            {((props.conversations?.length) != null) && props.conversations.length > 0
+            {((conversations?.length) != null) && conversations.length > 0
               ? (
-                  props.conversations.map((conversation: Conversation) => (
+                  conversations.map((conversation: Conversation) => (
                 <div key={conversation.id} className='conversation'>
                   <div className='conversation-header'>
-                    <h2>Conversation with {conversation.buyer_id}</h2>
+                    <h2>Conversation with {conversation.buyerId === user?.sub ? conversation.sellerId : conversation.buyerId}</h2>
```

Comment:
Mind making this say the user's name instead of their ID. It looks kind of weird in the UI.

We can call `GET /user/:id` to get their info.
![image](https://github.com/CPS-630/project/assets/56979977/5cc184dc-dc9a-4a85-9a67-dfe83f67db39)